### PR TITLE
add reduxjs/toolkit

### PR DESCRIPTION
**Pull request by Fourteen98**
### New Features: 
- Configured redux to project
- Implemented book actions and reducers.
- Implemented categories actions and reducers.
- Implemented redux store with configureStore

### CODE REVIEW

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Fourteen98,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
Kindly follow the instructions in the requirements and set up necessary methods and store reducers

- [x] You still need to set up the store as listed in the requirements in this file: `src/redux/configureStore.js`

- [x] Right now you have empty books actions setup. You need set up the books action types, action creators and the reducer in this file `src/redux/books/books.js`

- [x] Right now you have empty categories actions setup. You need to set up the categories action actions types, creators and the categories reducer in this file `src/redux/categories/categories.js`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const REMOVED_BOOK = 'REMOVE_BOOK';
+const ADD_BOOK = 'ADD_BOOK';
+
```
Comment (julie-ify) [inline src/redux/books/books.js]: - [x] You are required to use the ducks' pattern for your action types. See example below:

  ```
  const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
  const ADD_BOOK = 'bookstore/books/ADD_BOOK';
  ```

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```
Comment (julie-ify) [inline src/redux/categories/categories.js]: - [x] Kindly use ducks' pattern for your action types:

  ```
  const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
  ```

**Review by julie-ify — CHANGES_REQUESTED** [review]
Hi @Fourteen98,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @Fourteen98 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**